test/unit/dom/SVGElement.js

```diff
@@ -0,0 +1,46 @@
+import canPatchNativeAccessors from '../../../src/util/can-patch-native-accessors';
+import create from '../../lib/create';
+
+describe('dom: SVGElement', function () {
```

Comment:
airbnb was disabled for tests in namedslot
I'll turn it on and fix linting errors before merging this PR